---
layout: post
image: /images/mSphere.png
link: http://msphere.asm.org/content/3/3/e00220-18
title: "Our collaborative work with the Arnone Lab has been published in ASM mSphere!"
journal: "ASM mSphere"
year: 2018
date: 2018-06-14
category: post
---
It is essential that cells orchestrate gene expression for the specific
niche that they occupy, and this often requires coordination of the expression of
large sets of genes. There are multiple regulatory systems that exist for modulation
of gene expression, including the adjacent-gene coregulation of the rRNA and ribosome
biogenesis and ribosomal protein families. Both gene families exhibit a nonrandom
genomic distribution, often clustered directly adjacent to another member
of the same family, which results in a tighter transcriptional coordination among adjacent
paired genes than that of the unpaired genes within each regulon and can
result in a shared promoter that coordinates expression of the pairs. This nonrandom
genomic distribution has been seen in a few functionally related gene families,
and many of these functional pairings are conserved across divergent fungal lineages.
To date, the significance of these observations has not been extended in a
systematic way to characterize how prevalent the role of adjacent-gene coregulation
is in transcriptional regulation. In the present study, we systematically analyzed the
transcriptional coherence of the functional pairs compared to the singletons within
all gene families defined by the Gene Ontology Slim designation, using Saccharomyces
cerevisiae as a model system, finding that clusters exhibit a tighter transcriptional
correlation under specific contexts. We found that the longer a functional pairing is
conserved the tighter its response to broad stress and nutritional responses, that
roughly 25% of gene families exhibit a nonrandom genomic distribution, and that
many of these clusters are conserved. This suggests that adjacent-gene coregulation
is a widespread, yet underappreciated, transcriptional mechanism.
